wagtail/images/views/multiple.py

```diff
@@ -40,13 +40,21 @@ def get_upload_form_class(self):
     def get_edit_form_class(self):
         return get_image_multi_form(self.model)
 
+    def get_edit_object_response_data(self):
+        data = super().get_edit_object_response_data()
+        data["duplicate"] = self.is_duplicate
```

Comment:
It's probably a little clearer to determine whether it's a duplicate here, rather than setting it on the view and then retrieving it.